routes/feed.js

```diff
@@ -0,0 +1,37 @@
+const Feed = require('feed')
+const striptags = require('striptags')
+
+module.exports = function feedHandler (req, res, next) {
+  let feed = new Feed({
+    title: 'Electron',
+    description: 'Build cross platform desktop apps with JavaScript, HTML, and CSS.',
+    id: 'http://electronjs.org/',
+    link: 'http://electronjs.org/',
+    generator: 'Electron website',
+    feedLinks: {
+      json: 'https://electronjs.org/feed.json',
```

Comment:
Is `/feed.json` and `/feed.json` a convention? Or can we name it anything we want? If we can name it anything, I would prefer `blog.xml`, as that would free us up to have a pattern for things like `/releases.xml` in the future.

Reply:
Yes, we can name it anything we want.